src/util.js

```diff
@@ -87,12 +90,45 @@ export function serializeForm(form) {
 				values = [values];
 			}
 			values.forEach(value => {
-				let param = [name, value].map(encodeURIComponent).join("=");
-				params.push(param);
+				params.setParameter(name, value);
 			});
 		}
 		return params;
-	}, []).join("&");
+	}, params).serialize();
+}
+
+class FormDataParams {
+	constructor() {
+		this.formData = new FormData();
+	}
+
+	setParameter(name, value) {
+		this.formData.append(name, value);
+	}
+
+	serialize() {
```

Comment:
Naming is hard. Originally I had named this `toFormBody()`, but I changed that because for `GET` forms the result of `serializeForm` goes in the URI, not in the body. 

I'm not against "collate", but it is an obscure enough word that I have to look up its definition every time I see it.

Reply:
As discussed elsewhere, perhaps we should just use a getter:

```javascript
    constructor() {
        this._data = [];
    }

    get data() {
        return this._data.join("&");
    }
```